### DISCUSSION TITLE
zh/guide/drivers/Trainbit

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#207] [Announcements]
# zh/guide/drivers/Trainbit

Trainbit 官网：https://trainbit.com/ AUSHELLPORTAL ，Apikey 进入 Trainbit网页后，按F12打开开发者模式 在网络选项中随便打开一个刷新出来的数据，然后找到带有Cookie选项的都可以 填写参数对应的值即可 AUSHELLPORTAL：.AUSHELLPORTAL ` .AUSHELLPORTA...

https://alist.nn.ci/zh/guide/drivers/Trainbit.html

**Comment by bsgn2023** [comment]
新版上传显示“找不到该项目”，是否有解决方法，哪个版本通用所有盘的上传问题？

**Comment by simplove** [comment]
默认是这种http的链接下载，浏览器点了没反应，要复制出来才可以。是因为不是HTTPS的原因。。。这个要如何强制让它是https了
http://tb28.trainbit.com:8080/files/158477

**Reply by simplove** [reply]
试了https会报错，看来只能用“复制链接”了

**Comment by simplove** [comment]
Dear users, we would like to inform you that on November 1, 2023, all Trainbit servers will be shut down forever. Please back up your files by this date. From this date onwards, it will not be possible to access the site and files.
尊敬的用户，我们谨通知您，2023年11月1日起，所有Trainbit服务器将永久关闭。 请在此日期之前备份您的文件。 从即日起，将无法访问该网站和文件。

没得玩了，这个可以去掉了。
